api_compatibility_policy.md

```diff
@@ -29,35 +25,96 @@ A backwards incompatible change would be a change that requires a user to update
 existing instances of these CRDs in clusters where they are deployed and/or cause them
 to change their CRD definitions to account for changes in the above.
 
-## Alpha, Beta and GA
+## CRD API Versions
+
+The `apiVersion` field in a Tekton CRD determines whether the overall API (and its default behaviors) is considered to be in `alpha`, `beta`, or `GA`. The use of `alpha`, `beta` and `GA` (aka `stable`) follows the corresponding Kubernetes [API stages definition](https://kubernetes.io/docs/reference/using-api/#api-versioning) and its [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/).
```

Comment:
I think saying we follow k8s deprecation policy is a bit confusing actually (the main differences I see are the ability to mark a beta api as deprecated immediately, rather than waiting 9 months in addition to the 9 months required to remove it) and our feature flags policy.

Reply:
ok, yeah the k8s deprecation policy also includes things like cli flags that are not relevant to us at the moment